processor/groupbytraceprocessor/find_leak_test.sh

```diff
@@ -0,0 +1,8 @@
+# /bin/bash
+# Run this script only from current directory!
+
+go test -c -o tests
+for test in $(go test -list . | grep -E "^(Test|Example)");
+  do ./tests -test.run "^$test\$" &>/dev/null || echo "$test failed";
+done
+rm tests
```

Comment:
Took this script from goleak repo. It is extremely convenient to find leaky test. But still, do we need it here and should I copy the license?

Reply:
How accurate is it? Would it have reliably found this leak? If we are copying the code, it should come with the same license header.

Reply:
That said, would you be able to make it work with all other modules in the repo? Would be a good addition to the repo itself, I believe.

Reply:
Generally, as I understood it just compares goroutines running before all tests and after all tests in a module. Now the tests fail with unbuffered channel, so it's really helpful here (also helped to find minor leak in shutdown). I am curious about stability of this method and there is a good point in discussion in Jaeger repo that, unfortunately, the main test must be put in every package manually. From my perspective, it can be helpful in components that use goroutines for servers or async processing. And don't see any point to use it otherwise

Well, the script is copied not entirely, but I admit that we have to put the license

Reply:
I think I would prefer to leave it out for now from this PR. Instead, would you open an issue so that we can discuss a better, broader solution? Perhaps there's a linter out there that we can make use of?

Reply:
Agreed